src/index.js

```diff
@@ -106,7 +106,7 @@ const deprecationWarnings = ({steps}) => {
 }
 
 const isSMSCountryCodeValid = (smsNumberCountryCode) => {
-  const isCodeValid = countries.map(([code]) => code).includes(lowerCase(smsNumberCountryCode))
+  const isCodeValid = isSupportedCountry(upperCase(smsNumberCountryCode))
```

Comment:
Seems off a bit, IMO `isSupportedCountry` itself should eitgher lower-case or upper-case when comparing.

Reply:
Hmmm...yeah. `isSupportedCountry` actually is case insensitive when checking so forcing the country code to be uppercase is not necessary.